script.js

```diff
@@ -253,26 +255,40 @@ function startAlgo(event) {
     target.innerHTML = "Reset";
     target.classList.remove("button-primary");
   } else {
+    table.innerHTML = "";
     target.innerHTML = "Start";
     target.classList.add("button-primary");
     tryToDraw();
     return;
   } 
-  startLooping();
+  startLooping(0);
 }
-
-function startLooping() {
+function fillTable(stepNumber) {
+  
+  var row = table.insertRow(stepNumber);
+  var cell1 = row.insertCell(0);
+  var cell2 = row.insertCell(1);
+  var cell3 = row.insertCell(2);
+  var cell4 = row.insertCell(3);
+  var cell5 = row.insertCell(4);
+
+  cell1.innerHTML = a;
+  cell2.innerHTML = b;
+  cell3.innerHTML = midpoint;
+  cell4.innerHTML = f(midpoint) * f(a);
+  cell5.innerHTML = f(midpoint) * f(b);
+  console.log (a + " " + b + " " + midpoint + " " + f(midpoint) * f(a) + " " + f(midpoint) * f(b) );
```

Comment:
You can put this values in an array and iterate on it instead of making cell1, cell2, cell..